src/Symfony/Bundle/FrameworkBundle/Translation/TranslationLoader.php

```diff
@@ -63,4 +64,18 @@ public function loadMessages($directory, MessageCatalogue $catalogue)
             }
         }
     }
+
+    /**
+     * Loads translation messages from a directory to the catalogue.
+     *
+     * @param string           $directory the directory to look into
+     * @param MessageCatalogue $catalogue the catalogue
+     *
+     * @deprecated since 3.4 will be removed in 4.0. Use read instead.
+     */
+    public function loadMessages($directory, MessageCatalogue $catalogue)
+    {
+        $this->read($directory, $catalogue);
+    }
```

Comment:
same as https://github.com/symfony/symfony/pull/23667/files#r130601109, no need for deprecating the method if class deprecated first